tests/test-dirs/type-enclosing/jane-street.t/layouts.ml

```diff
@@ -25,10 +25,20 @@ let poly_client2 x = poly2 x
 let poly_client3 x = poly3 x
 let poly_client4 x = poly4 x
```

Comment:
Can we show that non-constant sort variables are not printed? Something along the lines of `let f x = x` perhaps

Reply:
Sure, done.